src/pyfme/environment/isa.py

```diff
@@ -0,0 +1,132 @@
+# coding: utf-8
+"""ISA functions.
+"""
+
+from math import exp
+
+# Constants
+R_a = 287.05287  # J/(Kg·K)
+g0 = 9.80665  # m/s^2
+
+# From: https://en.wikipedia.org/wiki/U.S._Standard_Atmosphere
```

Comment:
Perhaps we should not duplicate it, but at least the table with the layers should be in the docstring instead of the multiline comment I think.